skt/executable.py

```diff
@@ -334,6 +404,14 @@ def cmd_run(cfg):
 
     save_state(cfg, {'retcode': retcode})
 
+    if retcode != 2:
```

Comment:
Silly nitpick.  What is retcode 2?

Reply:
Just an agreement between us developers - 0 for success, 1 for failure and 2 for infrastructure errors.

Reply:
Ok.  Maybe someday we can wrap that with something more informational..